Dockerfile

```
@@ -0,0 +1,61 @@
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+# Fail the layer if any command in a pipe fails (e.g., `yes | sdkmanager ...`)
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)